fixtures/fixtures.js

```diff
@@ -55,12 +55,14 @@ var Fixtures = {
         ]
     },
     voucherStatus: {
-        "success":
-            {"success":["RVNT12345678"], "fail":[], "invalid":[]},
-        "fail":
-            {"success":[], "fail":["FAL11111111"], "invalid":[]},
-        "invalid":
-            {"success":[], "fail":[], "invalid":["INV1"]},
+        "success": [{"message": 'Voucher is valid'}],
+        "warning": [
+            {"warning": "You have already submitted voucher code :code."},
```

Comment:
I don't think it would be confusing. Fixtures are fixed examples of the data. If you want to - call it XXX00000 - but I think it should be in the same format as the real thing. The rest of the fixtures have arbitrary codes.